book/en/src/by-example/app_priorities.md

```diff
@@ -4,7 +4,7 @@
 
 The `priority` argument declares the static priority of each `task`.
 
-For Cortex-M, tasks can have priorities in the range `1..=(1 << NVIC_PRIO_BITS)`
+For Cortex-M, tasks can have priorities in the range `1..(1 << NVIC_PRIO_BITS)`
```

Comment:
In Rust, `..=` indicates it is an inclusive range

Reply:
https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html

Reply:
If I recall correctly that is intentional, we can provide one more priority level by disabling the interrupts globally.

Reply:
I was not aware of the rangeinclusive notation. Thx.

Reply:
Fixed.